### PR TITLE
Fix:Set customJsonType lost /

### PR DESCRIPTION
**Pull request by cage1618**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
It seems this is targeting the wrong branch. Can you target `master` instead?

**Comment by cage1618** [comment]
> It seems this is targeting the wrong branch. Can you target `master` instead?

done

**Comment by cage1618** [comment]
> Does this PR really need to touch 108 files?

I don't understand what is touch 108 files

**Comment by mcollina** [comment]
Can you please add a unit test?

**Comment by jsumners** [comment]
> > Does this PR really need to touch 108 files?
> 
> I don't understand what is touch 108 files

This PR originally modified 108 files.

File: lib/reply.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ Reply.prototype.send = function (payload) {
 
           // We ensure we set the header to the proper JSON content-type if necessary
           // (e.g. 'application/hal+json' instead of 'application/json')
-          this[kReplyHeaders]['content-type'] = CONTENT_TYPE.JSON.replace('/json', customJsonType)
+          this[kReplyHeaders]['content-type'] = CONTENT_TYPE.JSON.replace('/json', `/${customJsonType}`)
```
Comment (jsumners) [inline lib/reply.js:168]: What if `customJsonType === '/my+json'?

**Comment by mcollina** [comment]
@cage1618 can you add a unit test for this?

**Pull request closed without merging**